### PR TITLE
Stream Deck Plus support + some fixes

### PR DESCRIPTION
**Pull request by K900**
Add new message types for all the new Stream Deck+ stuff, fix color parsing for the latest versions of the app, update docs links

### CODE REVIEW

**Comment by kristofferpeterhansel** [comment]
Hehe. Very similar to the changes in my fork.

The color issue is that there is now an alpha channel in the color. So a fourth number (which can also be seen in the sample on the site: https://docs.elgato.com/sdk/plugins/registration-procedure#info-parameter).

My hack so far is just to accept the extra two bytes but ignoring them. The documentation does not seem very clear if it will always be there or not.

**Comment by K900** [comment]
@mdonoughe can you take a look?

File: src/registration.rs [hunk 1]
```diff
@@ -156,7 +156,7 @@ pub struct RegistrationInfo {
     pub plugin: RegistrationInfoPlugin,
     pub device_pixel_ratio: u8,
     pub devices: Vec<RegistrationInfoDevice>,
-    pub colors: UserColors,
+    // pub colors: UserColors,
```
Comment (mdonoughe) [inline src/registration.rs]: The colors should stay.

**Comment by K900** [comment]
Added proper RGBA colors support, also added docs for the new SD+ stuff.

**Comment by K900** [comment]
I finally got around to fixing the very obvious issues in my plugin that uses this, so https://gitlab.com/K900/streamdeck-now-playing is now up if anyone is interested in a code sample.

**Review by mdonoughe — APPROVED** [review]

**Comment by K900** [comment]
Re-ran rustfmt, no other changes.

**Pull request merged**